app/assets/javascripts/views/searches/preview.js

```diff
@@ -6,7 +6,7 @@
   this.Terraling.Searches = this.Terraling.Searches || {};
 
   var searches = this.Terraling.Searches;
-  
+
```

Comment:
Turn off your IDE auto-spacing settings, or any PR will have pages and pages with this kind of silly changes.